content/docs/introducing-jsx.md

```diff
@@ -175,10 +188,13 @@ const element = {
 };
 ```
 
-These objects are called "React elements". You can think of them as descriptions of what you want to see on the screen. React reads these objects and uses them to construct the DOM and keep it up to date.
+Deze objecten worden "React elementen" genoemd.
+Je kunt ze zien als beschrijvingen van wat je wilt zien op het scherm.
+React leest deze objecten en gebruikt ze om het DOM op te bouwen and bij te werken.
 
-We will explore rendering React elements to the DOM in the next section.
+We zullen het renderen van React elementen naar het DOM in het volgende hoofdstuk verder bekijken.
```

Comment:
```suggestion
We zullen het renderen van React elementen naar de DOM in het volgende hoofdstuk verder bekijken.
```
Even though the Dutch word "model" would require a "het" article, I think "de DOM" sounds better.

I think this principle applies here:

> 1. Leenwoorden worden in beginsel de-woorden (die ook binnen de gewone Nederlandse woordenschat veruit in de meerderheid zijn), tenzij er een reden is om er het-woorden van te maken. 

Source: https://taaladvies.net/taal/advies/tekst/40/woordgeslacht_van_engelse_leenwoorden_algemeen/

Reply:
As I wrote before:
> Well, it's an abbreviation for Document Object Model and it's "het model". But I always say "de DOM" as well 😄
> The Dutch Wikipedia page about the DOM also uses "het DOM": https://nl.wikipedia.org/wiki/Document_Object_Model

Reply:
I think we need to just make a decision. Either solution has a plausible explanation, I think.

Reply:
I always say "de DOM" as "alliteratie" (to stress). But I think it is wrong to use in general, even though it is wrong according to taaladvies: https://taaladvies.net/taal/advies/tekst/39/woordgeslacht_van_afkortingen_algemeen/. I don't agree that Document Object Model are "leenwoorden". MDN also uses "het DOM" (oh... and "de DOM" once): https://developer.mozilla.org/nl/docs/Web/API/Document_Object_Model

Reply:
@rickbeerendonk I don't see what word category it would fall under if not under "leenwoorden".  To use general Dutch abbreviation rules for it, makes no sense to me. What MDN do is translate the whole construct from "Domain Object Model" to "Document Object Model", a construct that makes some sense in Dutch. Then, it makes sense to follow general abbreviation rules and since the Dutch "Model" is gender-neutral, it makes sense to use "het".

Personally, I don't think it makes sense to translate the term at all, in which case I think it should be "de DOM".

What do you guys think?
@rickbeerendonk @Keraito @ronderksen

Reply:
I think we should follow other translations, like MDN and Wikipedia, which use "het DOM".